network-api/networkapi/wagtailpages/templates/wagtailpages/research_library_page.html

```diff
@@ -1,35 +1,31 @@
 {% extends "./research_base.html" %}
 {% load i18n static wagtailcore_tags wagtailimages_tags %}
-
-
 {% block research_hub_content %}
     <div>
         {% include './fragments/research_breadcrumb.html' with breadcrumb_list=breadcrumbs %}
         <h1>{{ page.title }}</h1>
     </div>
-
-    <div class="large:tw-w-160 large:tw-h-64 tw-mt-12 tw-mb-8 large:tw-mb-0 large:-tw-ml-12 large:tw-p-12 large:tw-bg-gray-05" >
+    <div class="large:tw-w-160 large:tw-h-64 tw-mt-12 tw-mb-8 large:tw-mb-0 large:-tw-ml-12 large:tw-p-12 large:tw-bg-gray-05">
         {# SEARCH BAR #}
         {% comment %}
       The page url is necessary in the form to that the filter anchor link is not carried forward.
       Say on mobile the user used the "Filter" anchor link to get to the filter section,
       that adds the `#filter` part to the URL. Submitting the form in that stage without the
       explicit URL would carry the anchor link forward. That would mean the view is scrolled to the
       filter section again upon page reload. This seems undesireable.
-    {% endcomment %}
-        <form action="{% pageurl page %}" method="get" accept-charset="utf-8" id="search-form">
+        {% endcomment %}
+        <form action="{% pageurl page %}"
+              method="get"
+              accept-charset="utf-8"
+              id="search-form">
```

Comment:
I decided to separate the search form from the filter form. The rendering of the filter form gets very messy if adding the search field, as well as the html here. The functionality should be the same, though.